site/src/components/PageSidebar.tsx

```diff
@@ -187,11 +187,7 @@ const SidebarPage: VFC<SidebarPageProps> = ({
               setSelectedAnchorElement(ref);
             }
           }}
-          onClick={() => {
```

Comment:
The default behavior of `Link` components is to scroll to top: https://nextjs.org/docs/api-reference/next/link#disable-scrolling-to-the-top-of-the-page.

This prevents scroll to top if a user is in the docs or spec set of pages.

Reply:
@CiaranMn I think this is the definite answer to the behavior we were observing here.